platforms/core-configuration/core-serialization-codecs/src/main/kotlin/org/gradle/internal/serialize/codecs/core/UnsupportedTypesCodecs.kt

```diff
@@ -145,7 +145,7 @@ fun BindingsBuilder.unsupportedTypes() {
     bind(unsupported<BuildService<*>>())
 
     // Gradle implementation types
-    bind(unsupported<DefaultServiceRegistry>())
+    bind(unsupported<ServiceLookup>())
```

Comment:
I wondered if there were really no `ServiceLookup` implementations that could be supported by the configuration cache, but tests say we are good, so :+1:  from me.